doc/storage_data_plane_commands_auth.md

```diff
@@ -0,0 +1,74 @@
+# Storage Data Plane Commands Auth
+
+Azure operations can be divided into two categories - control plane and data plane. By running storage data plane commands, requests will be sent directly to Storage service instead of being sent to ARM(Azure Resource Manager). Hence additional credentials are required for successfully auth to Storage service.
```

Comment:
We usually call it mgmt-plane and data-plane, don't we?

Reply:
I want to follow the doc I shared in `See More` section: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/control-plane-and-data-plane

> Azure operations can be divided into two categories - control plane and data plane. This article describes the differences between those two types of operations.
> 
> You use the control plane to manage resources in your subscription. You use the data plane to use capabilities exposed by your instance of a resource type.